Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
```

Comment:
Ah, I just kept the name from the default value:
```
--socket-file|-n [arg]                 Use this file for the server socket (default ~/.rdm).
```
Would it be better to just use `rdm` instead?